CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.16)
+project(tetris CXX)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+find_package(SFML 2.5 REQUIRED COMPONENTS graphics window)
+
+add_executable(tetris
+    Source/DrawText.cpp
+    Source/GetTetromino.cpp
+    Source/GetWallKickData.cpp
+    Source/Main.cpp
+    Source/Tetromino.cpp)
+target_include_directories(tetris PRIVATE Source/Headers)
+target_link_libraries(tetris PRIVATE sfml-graphics sfml-window)
```

Comment:
Since you already specified 3.14+ as a requirement, please add the following at the end, so one does not have to spelunk in the binary directory for the executable:

```cmake
install(TARGETS tetris)
```

With this, one can just `cmake --install build --config Release --strip --prefix prefix` to get a stripped executable in the `prefix` directory.

Reply:
Good idea. Done.